scripts/generate_dump

```diff
@@ -1108,6 +1108,8 @@ collect_mellanox() {
       echo "HW Mgmt dump script $HW_DUMP_FILE does not exist"
     fi
 
+    # Temporarily collect information for debugging the issue that asic_table.json not rendered
+    save_file "/usr/share/sonic/templates/asic_table.j2" dump false
```

Comment:
this is a static file, why collect?

Reply:
regarding the issue [#13791](https://github.com/sonic-net/sonic-buildimage/issues/13791), one possible reason is that the template is not available when it is being rendered.
Even the probability is very slim, I would like to collect the template to rule out (or prove) it.
How do you think?

Reply:
It is static file, and show tech already collect the version, so we could check the file in that image. There is no need to collect here.

Reply:
ok. then closing it.